torch/csrc/distributed/c10d/reducer.hpp

```diff
@@ -538,7 +538,7 @@ class TORCH_API Reducer {
   std::unordered_map<size_t, std::string> param_names_;
   // Variable indices stored sequentially in order of when the gradient is ready
   // for the current backwards pass.
-  std::vector<int> grad_ready_order_indices_;
+  std::vector<size_t> grad_ready_order_indices_;
```

Comment:
verified signed to unsigned seems to be ok

Reply:
Fixed